Album/ViewControllers/ColorPickViewController.m

```diff
@@ -0,0 +1,40 @@
+//
+//  ColorPickViewController.m
+//  Album
+//
+//  Created by Airei Fukuzawa on 7/19/22.
+//
+
+#import "ColorPickViewController.h"
+#import "FCColorPickerViewController.h"
+
+@interface ColorPickViewController () <FCColorPickerViewControllerDelegate>
+@property (nonatomic, strong) UIColor *color;
+@end
+
+@implementation ColorPickViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
```

Comment:
I'd remove this from all ViewCopntrollers

Reply:
Removed!